src/components/MovieList/MovieList.jsx

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable react/jsx-key */
+import React, { useEffect } from "react";
+import PropTypes from "prop-types";
+import Film from "../Film/Film";
+
+import "./style.scss";
+
+const MovieList = ({ films, onGetilms, genre }) => {
+  useEffect(() => {
+    onGetilms();
+  }, []);
+
+  const changeGenre = arr => {
+    return arr.reduce((acc, item, index) => {
+      genre.forEach(elem => {
+        if (elem.id === item) {
+          acc[index] = elem.name;
+        }
+      });
+      return acc;
+    }, []);
+  };
+
+  return (
+    <section className="movie-list">
+      {films &&
```

Comment:
not needed to check for `films` before
the `films` is required

Reply:
changed